admin_guide/scheduling/pod_placement.adoc

```diff
@@ -293,26 +293,29 @@ endif::[]
 . Create a project object that includes the
 `scheduler.alpha.kubernetes.io/node-selector` annotation and labels.
 +
+[source,yaml]
 ----
-{
-    "kind": "Namespace",
-    "apiVersion": "v1",
-    "metadata": {
-        "name": "ns1",
-        "annotations": {
-            "scheduler.alpha.kubernetes.io/node-selector": "env=test,infra=fedora" <1>
-        }
-    },
-    "spec": {},
-    "status": {}
-}
-
+apiVersion: v1
+kind: Namespace
+metadata
+  name: ns1
+  annotations: 
+    scheduler.alpha.kubernetes.io/node-selector: env=test,infra=fedora <1>
+spec: {},
+status: {}
 ----
 +
 <1> Annotation to create the labels to match the project label selector. Here, the key/value labels are `env=test` and `infra=fedora`.
++
+[NOTE]
+====
+When using the *Pod Node Selector* admission controller, you cannot use the `oc new-project` command as
```

Comment:
Not `oc new-project`, but oc adm your-project for setting project node selector.